try {
    this.addEventListener('fetch', () => {});
} catch (error) {
    console.log(error);
}
